.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -0,0 +1,72 @@
+name: Wheels
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CIBW_BUILD: 'cp37-* cp38-* cp39-*'
+
+  # MacOS specific build settings
+  CIBW_BEFORE_ALL_MACOS: |
+    brew uninstall --force oclint
+    brew install gcc libomp
```

Comment:
Maybe a silly question, but do we only technically need this for the mac builds? But maybe it's easier to keep everywhere?

Reply:
Well spotted. I thought I had pulled all of those --- will remove it. Cheers.